src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -158,63 +157,6 @@ func (a *AdminAPI) sendOne(method, path string, body, into interface{}) error {
 	return maybeUnmarshalRespInto(method, url, res, into)
 }
 
-// sendAll sends a request to all URLs in the admin client. The first successful
```

Comment:
Does this PR also remove the need for `NewHostClient`, and in redpanda/config/config.go, removes the need for the `--host` flag?

Reply:
`--host` looks like it could be used either for specifying a host, or for specifying an API endpoint within a single node (when an integer is used).  The latter use case still makes sense even when we can redirect between nodes.

Reply:
:+1: re-double checked again, for some reason I misremembered why I added that flag, and thought it was related to this -- it isn't.